### DISCUSSION TITLE
Allow access to WiFi with entry page

### DISCUSSION THREAD

**Discussion by outdoorbits** [davesteele/comitup#229] [Ideas]
While traveling a few times I had the following problem: The WiFi was not protected by password but there was a entry page, asking for username and password. In this hotels my little-backup-box had to stay offline.
It was perfect, if comitup would ask not only for the WiFi password but also/alternative for the address of the entry page and two text fields for this portal page and fill this into that pages form.
Is this possible at all?
Thank you so much for comitup!

**Comment by davesteele** [comment]
That's not practical for use across hotels. But, if you add a second wifi device, and operate in appliance mode, comitup will let you use your phone to get past the hotel login.

**Comment by outdoorbits** [comment]
That sounds like a perfect solution, thank you!
